src/platform/i386/boot_comp.c

```diff
@@ -134,16 +139,21 @@ kern_boot_thd(struct captbl *ct, void *thd_mem, void *tcap_mem)
 }
 
 void
-kern_boot_comp(void)
+kern_boot_comp(const u32_t cpu_id)
 {
 	int            ret = 0, nkmemptes;
-	struct captbl *ct;
 	unsigned int   i;
 	u8_t *         boot_comp_captbl;
-	void *         thd_mem, *tcap_mem;
 	pgtbl_t        pgtbl     = (pgtbl_t)chal_va2pa(&boot_comp_pgd), boot_vm_pgd;
 	u32_t          hw_bitmap = 0xFFFFFFFF;
 
+	if (cpu_id > 0 ){
```

Comment:
Must be `if (cpu_id > 0) {`.. Spacing correction only!

Reply:
Sorry for this. I will double check with the CSG.

Reply:
dito